### PR TITLE
feature: add Prod stage and release

### PR DESCRIPTION
**Pull request by Trishisingh**

### CODE REVIEW

File: build/azDevOps/azure/air-infrastructure-data.yml [hunk 1]
```diff
@@ -106,154 +106,101 @@ stages:
...
-  #       pool:
-  #         vmImage: $(pool_vm_image)
-
-  #       environment: ${{ variables.domain }}-prod
-
-  #       strategy:
-  #         runOnce:
-  #           deploy:
-  #             steps:
+#   yamllint disable rule:comments-indentation
```
Comment (ElvenSpellmaker) [inline build/azDevOps/azure/air-infrastructure-data.yml:109]: If you just put the comment on the line before this shouldn't be a problem I think?

File: build/azDevOps/azure/air-infrastructure-data.yml [hunk 2]
```diff
@@ -106,154 +106,101 @@ stages:
...
+                    TF_VAR_name_domain: $(domain)
+                    TF_VAR_stage: $(Environment.ShortName)
+                    
+  - stage: Release
+    dependsOn:
+      - Build
+      - Prod
+    condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/master'), eq(variables['create_release'], 'true'))
+    variables:
+      - group: amido-stacks-infra-credentials-nonprod
```
Comment (ElvenSpellmaker) [inline build/azDevOps/azure/air-infrastructure-data.yml:175]: This is prod?

File: build/azDevOps/azure/air-infrastructure-data.yml [hunk 3]
```diff
@@ -106,154 +106,101 @@ stages:
...
+                    TF_VAR_name_company: $(company)
+                    TF_VAR_name_project: $(project)
+                    TF_VAR_name_domain: $(domain)
+                    TF_VAR_stage: $(Environment.ShortName)
+                    
+  - stage: Release
+    dependsOn:
+      - Build
+      - Prod
+    condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/master'), eq(variables['create_release'], 'true'))
```
Comment (ElvenSpellmaker) [inline build/azDevOps/azure/air-infrastructure-data.yml:173]: `master` or `main`, you have both in this PR (maybe we should put this into variables). - Let's speak further

File: build/azDevOps/azure/air-infrastructure-data.yml (hunk 1 above)
Comment (Trishisingh) [reply build/azDevOps/azure/air-infrastructure-data.yml]: udated

File: build/azDevOps/azure/air-infrastructure-data.yml (hunk 2 above)
Comment (Trishisingh) [reply build/azDevOps/azure/air-infrastructure-data.yml]: updated this to prod

File: build/azDevOps/azure/air-infrastructure-data.yml (hunk 3 above)
Comment (Trishisingh) [reply build/azDevOps/azure/air-infrastructure-data.yml]: for now have updated this to main, but lets speak to see what do we usually do

File: build/azDevOps/azure/air-infrastructure-data.yml [hunk 4]
```diff
@@ -106,154 +106,102 @@ stages:
...
-  #       value: prod
-  #     - name: dns_base_domain # Available in Terraform Output, but not yet enabled to export as pipeline vars
-  #       value: prod.amidostacks.com
-  #     - name: core_environment
-  #       value: prod
-  #   jobs:
-  #     - deployment: AppInfraProd
-  #       pool:
-  #         vmImage: $(pool_vm_image)
+ #   yamllint disable rule:comments-indentation
```
Comment (ElvenSpellmaker) [inline build/azDevOps/azure/air-infrastructure-data.yml:109]: I'm  not sure we need this if we move the comments below to above the lines?

Comment (Trishisingh) [reply build/azDevOps/azure/air-infrastructure-data.yml]: thanks for the help, all updated now.

**Review by ElvenSpellmaker — APPROVED** [review]

**Pull request merged**